### ISSUE TITLE
Nextjs  中 createFromIconfontCN 报错 

### ISSUE THREAD

**Issue by LengYXin** [ant-design/ant-design-icons#589]
createFromIconfontCN 会转换为  @ant-design/icons/lib/icons/createFromIconfontCN

<img width="832" alt="image" src="https://github.com/ant-design/ant-design-icons/assets/19631404/28e8b45a-deb1-4f80-9f46-472e6d7a3bd8">

无其他代码只有 导入创建
```

import { createFromIconfontCN } from '@ant-design/icons';
const IconFont = createFromIconfontCN({
  scriptUrl: '//at.alicdn.com/t/font_8d5l8fzk5b87iudi.js',
});
console.log("LENG ~ IconFont:", IconFont);
```

Nextjs 13.4.12

**Comment by wwz223** [comment]
有解决吗 @LengYXin

**Comment by afc163** [comment]
Fixed in `@ant-design/icons@5.4.0`: https://github.com/ant-design/ant-design-icons/pull/653
